build-system/runner/src/org/ampproject/AmpCommandLineRunner.java

```diff
@@ -134,10 +127,6 @@ public static void main(String[] args) {
         runner.typecheck_only = true;
       } else if (arg.contains("FORTESTING=true")) {
         runner.is_production_env = false;
-      } else if (arg.contains("PSEUDO_NAMES=true")) {
-        runner.pseudo_names = true;
```

Comment:
This is still needed.

Reply:
I thought it was only used by single-pass. Reinstated in 5ac8a319e31151b196330563b3aec41257ec4cb4